src/models/Activity.ts

```diff
@@ -1,42 +1,25 @@
-import { ActivityType, StationType } from '../assets/enums';
-
+import { ActivityType } from '@dvsa/cvs-type-definitions/types/v1/enums/activityType.enum';
```

Comment:
Is there a benefit to redefining the lists here, rather than adding the array directly to the appropriate schema?

Reply:
Pushed up a change, I think its best of both worlds but let me know thoughts.

Reply:
Yes I think this approach is clean, thanks.